src/eredis.app.src

```diff
@@ -1,9 +1,6 @@
 {application,eredis,
              [{description,"Erlang Redis Client"},
-              {vsn,"1.2.0"},
-              {modules,[eredis,eredis_client,eredis_parser,eredis_sub,
```

Comment:
I'm sure this isn't required. It's most surely created by `rebar3` when required.

Reply:
Yeah, we can do away with this.

Reply:
Yes, we can surely do away with it.